recipes/openmm-atmmetaforce-plugin/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "openmm-atmmetaforce-plugin" %}
+{% set version = "0.2.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cf7975639c285ed0fab441c2ac36e3cbf6e499256a708240d4acfeb562939bae
+
+build:
+  number: 0
+  skip: true  # [not linux64]
+  skip: true  # [cuda_compiler_version in (undefined, 'None')]
+
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy
+    - swig
+    - openmm
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux]
+    - cmake
+    - doxygen
+
+  host:
+    - python
+    - openmm
+    - fftw
+    - swig
+    - pthread-stubs
+    - lxml
+    - numpy
+    - cython
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
+    
+  run:
+    - python
+    - openmm
+    - fftw
+    - numpy
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - ocl-icd-system  # [linux]
+    - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
+    - ocl_icd_wrapper_apple      # [osx and opencl_impl == 'khronos']
```

Comment:
Please double check these requirements. I searched your plugins source code and didn't see any mention of numpy, fftw, or cython. Also, the build section is for build tools, host is for libraries or tools that must match the target platform when cross-compiling, and the run section is for python dependencies.

Do you need doxygen? I doesn't look like your are shipping docs pages.

Reply:
Thank you. I have revised the build and host requirements as suggested and following the examples.